src/components/learner-credit-management/AssignmentTableCancel.jsx

```diff
@@ -122,7 +123,12 @@ const AssignmentTableCancelAction = ({
   return (
     <>
       <Button variant="danger" iconBefore={DoNotDisturbOn} onClick={openModal}>
-        {`Cancel (${totalToCancel})`}
+        <FormattedMessage
+          id="lcm.budget.detail.page.catalog.tab.course.card.cancel"
```

Comment:
@jajjibhai008 is this lcm budget detail page or assignment page?

Reply:
assignment flow is embedded in the form of components on the LCM budget detail page. So in short it's in the LCM budget detail page.

Reply:
@jajjibhai008 may be we should consider renaming the ids to assignments.page or something for better backtracking?